lib/index.js

```diff
@@ -1,16 +1,34 @@
-const { compress: _compress, decompress: _decompress } = require('bindings')('zstd');
+'use strict';
+const zstd = require('bindings')('zstd');
+const { promisify } = require('util');
+const { isArrayBufferView } = require('util/types');
 
+const _compress = promisify(zstd.compress);
+const _decompress = promisify(zstd.decompress);
 // Error objects created via napi don't have JS stacks; wrap them so .stack is present
 // https://github.com/nodejs/node/issues/25318#issuecomment-451068073
 
-exports.compress = async function compress(data) {
+exports.compress = async function compress(data, compressionLevel) {
+  const isUint8Array = isArrayBufferView(data) && data instanceof Uint8Array;
```

Comment:
JS instanceof is generally bad because it doesn't account for contexts/realms, why not just use the `isUint8Array` that gets exported from `util/types` already?